packages/clarity-decode/src/core.ts

```diff
@@ -0,0 +1,10 @@
+import { Data, Layout } from "clarity-js";
+
+export function expand(input: string): [string, string] {
+    let parts = input.split(Data.Constant.Dot);
```

Comment:
Suggestion: Write as [length, checkum] = input.split

Both shorter and simpler.

Reply:
Thanks, done.